subprojects/core/src/main/resources/org/gradle/initialization/removed-types.txt

```diff
@@ -19,6 +19,10 @@ org.gradle.api.internal.tasks.testing.junit.report.TestReporter
 org.gradle.api.internal.tasks.execution.TaskValidator
 org.gradle.api.artifacts.transform.ArtifactTransformDependencies
 org.gradle.internal.filewatch.FileWatcherFactory
+<<<<<<< HEAD
+org.gradle.api.tasks.incremental.IncrementalTaskInputs
+=======
 org.gradle.api.artifacts.transform.ArtifactTransform
 org.gradle.api.artifacts.transform.ArtifactTransformException
 org.gradle.api.artifacts.transform.VariantTransform
+>>>>>>> master
```

Comment:
You need to remove the conflict markers heres.

Reply:
Good catch